### PR TITLE
testing application with node 14

### LINKED ISSUE #10: Test Makiwich with Node 14
Let's make sure Makiwich works as expected with Node 14.

### PR DESCRIPTION
**Pull request by shabnomnom**
Testing the application with Node 14 since we are going to launch api-markers with this node version.
 closes #10

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,4 +2,5 @@ language: node_js
 node_js:
 - 10
```
Comment (tristen) [inline .travis.yml]: Should 10 just be dropped? or can we _just_ test 14?

**Comment by tristen** [comment]
👋 @shabnomnom @riastrad is Travis set up in this repo? I don't see your PR triggering any builds.

**Comment by riastrad** [comment]
Oh, weird. We saw the `.travis.yml` file and assumed this repo was all set. 

Taking a quick glance at https://github.com/mapbox/makiwich/pull/5, it looks like it was set up previously but isn't running now. We can dig into why it's no longer running before we get your review & merge. 👍

File: .travis.yml (hunk 1 above)
Comment (riastrad) [reply .travis.yml]: I'm not opposed to dropping 10 from our CI tests, but will let @shabnomnom weigh in as well.

**Comment by shabnomnom** [comment]
looks like that this repo is set up to run on `travis-ci.org` starting June 15 2021 `travis-ci-org` is ceased and we will need to migrate this repo to `travis-ci.com`. 

Screen Shot 2021-06-30 at 11 23 53 AM

**Comment by tristen** [comment]
@shabnomnom ah! if you visit https://travis-ci.com/github/mapbox/makiwich you can start the migration there. Then tt should just be a matter of pushing another commit to trigger things.

**Comment by shabnomnom** [comment]
@riastrad we briefly talked about challenges of upgrading Node modules. I see my build is not passing for node 14. Do you have any resources for this to help me with the next steps?

**Comment by riastrad** [comment]
👋 hiya @shabnomnom looks like you're running into compatibility issues with the `mapnik` dev dependency and the latest version of Node v14. Looking at the [CHANGELOG.md](https://github.com/mapnik/node-mapnik/blob/master/CHANGELOG.md#450), it looks like `mapnik` didn't upgrade to N-API & a universal binary until v4.5.0. Since `mapnik` is only a dev dependency used for tests in this project, you should be able to upgrade it to v4.5.0 without too much trouble.

**Comment by shabnomnom** [comment]
It worked! Thanks @riastrad!

File: readme.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Composites [Maki](https://mapbox.com/maki) icons with a map marker and returns an SVG.
 
-[![](https://api.travis-ci.org/mapbox/makiwich.svg?branch=master)](https://travis-ci.org/mapbox/makiwich)
+[![](https://api.travis-ci.com/mapbox/makiwich.svg?branch=master)](https://travis-ci.com/mapbox/makiwich)
```
Comment (tristen) [inline readme.md]:
```suggestion
[![](https://api.travis-ci.com/mapbox/makiwich.svg?branch=main)](https://travis-ci.com/mapbox/makiwich)
```

**Review by tristen — APPROVED** [review]
Just left a small commit suggestion. Otherwise looks great!

**Pull request merged**